testsuite/features/secondary/srv_restart.feature

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+# After clicking the button we wait until we dont see the text. This normally
+# means that the page was refreshed, but in case of a timeout or other error,
+# we included a manual refresh and a simple navigation step to make sure the UI
+# is indeed up and running after the restart.
+
+Feature: Restart the spacewalk services via UI
+
+  Scenario: Restart the SUSE Manager through the WebUI Admin option
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Manager Configuration > Restart"
+    And I check "restart"
+    And I click on "Restart"
+    And I wait until I see "SUSE Manager restarting" text
+    And I wait at most "300" seconds until I do not see "SUSE Manager restarting" text
+    And I refresh the page
+    Then I follow the left menu "Admin > Manager Configuration > Restart"
```

Comment:
`And`

Also, what's the purpose of this last `I follow the left menu "Admin > Manager Configuration > Restart"` ?
Maybe you are lacking another test starting with `Then` on line 19 ?

Reply:
Its a check that the server is up after the restart. I just navigate basically to another page to see that its up

Reply:
why not finish with `Then`? Is `And` necessary at the last line?

Reply:
actions (I do something) start with "When"